integration-tests/__tests__/globals.test.js

```diff
@@ -120,10 +120,11 @@ test('tests with no implementation', () => {
 
   writeFiles(TEST_DIR, {[filename]: content});
   const {stderr, status} = runJest(DIR);
-  expect(status).toBe(0);
+  expect(status).toBe(1);
 
   const {summary, rest} = extractSummary(stderr);
-  expect(rest).toMatchSnapshot();
+  const restWithoutStack = rest.slice(0, 343);
```

Comment:
Heh, seems somewhat arbitrary.

We should probably clean the stack up better instead of adding this hack

Reply:
Should I used the formatStackTrace from jest-message-utils when creating the error?

Reply:
No, it's called automatically.

As it's only an issue for the integration tests (as they used linked files instead of inside `node_module`), I think we should be good with just stripping that single line in https://github.com/facebook/jest/blob/d065e87a2322e1b4b70e7ba9de3c24aed7a2ea72/integration-tests/Utils.js#L162

Reply:
@SimenB I'm struggling with this one bit. `cleanupStackTrace` doesn't replace the stacktrace for the Env, but does for everything else.